include/deal.II/base/aligned_vector.h

```diff
@@ -292,16 +292,17 @@ namespace internal
      * Constructor. Issues a parallel call if there are sufficiently many
      * elements, otherwise work in serial. Copies the data from source to
      * destination and then calls destructor on the source. If the optional
-     * argument is set to true, the source is left untouched instead.
+     * argument copy_source is set to true, the source is left untouched
+     * instead.
```

Comment:
The argument is no longer optional now. Please update the text. (While you're in there, in the second line it should be "works" instead of "work" as well.)